src/templates/layouts/healthCareLocalFacility/HealthServices/HealthServices.tsx

```diff
@@ -0,0 +1,87 @@
+import { FormattedVAMCFacilityHealthServices } from '@/types/formatted/healthCareLocalFacility'
+import { ServiceLocation } from './ServiceLocation'
+
+export const HealthServices = ({
+  healthServices,
+}: {
+  healthServices: FormattedVAMCFacilityHealthServices[]
+}) => {
+  return (
+    <>
+      <h2
+        id="health-care-offered-here"
+        className="vads-u-font-size--xl vads-u-margin-top--5"
+      >
+        Health services offered here
+      </h2>
+      <p>Select a topic to learn more.</p>
+
+      <va-accordion>
+        {healthServices.map((service) => {
+          const id = service.name
+            .toLowerCase()
+            .replace(/\s+/g, '-')
+            .slice(0, 60)
+          const isVha = service.fieldFacilityLocatorApiId?.startsWith('vha_')
+          const hasLocationData =
+            service.locations?.[0]?.single?.fieldServiceLocationAddress
+
+          return (
+            <va-accordion-item
+              key={id}
+              data-label={service.name}
+              data-childlabel={service.fieldAlsoKnownAs}
+              data-template="facilities/facilities_health_service"
+              id={id}
+              class="facilities_health_service va-accordion-item"
+              subheader={service.fieldAlsoKnownAs}
+            >
+              <h3 slot="headline">{service.name}</h3>
+              <div id={`${service.entityBundle}-${service.entityId}`}>
+                {service.fieldCommonlyTreatedCondition && (
+                  <p className="vads-u-margin-bottom--2">
+                    Common conditions: {service.fieldCommonlyTreatedCondition}
+                  </p>
+                )}
+
+                {service.fieldTricareDescription ? (
+                  <div>{service.fieldTricareDescription}</div>
+                ) : service.description ? (
```

Comment:
`service.description` is an object because it is a wysiwyg field. This should either be selecting the `value` or `processed` values. I think that depends on the value of `format` but I'm not sure, I think we need to look at how other wysiwyg fields are rendered.

Reply:
If it's a wysiwyg field we always use processed; On the other hand, if this is rendering, `description` may not actually be a wysiwyg field.